marvel-client/main.go

```diff
@@ -0,0 +1,60 @@
+package main
+
+// marvel.NewClient()
+//client.GetCharacters()
+import (
+	"encoding/json"
+	"log"
+	"net/http"
+	"os"
+	"time"
+
+	"github.com/joho/godotenv"
+)
+
+func main() {
+	err := godotenv.Load()
+	if err != nil {
+		log.Fatal("Error loading .env file")
+	}
+	publicKey := os.Getenv("marvel_public_key")
+	privateKey := os.Getenv("marvel_private_key")
+
+	client := marvelClient{
+		publicKey:  publicKey,
+		privateKey: privateKey,
+		httpClient: &http.Client{
+			Timeout: 20 * time.Second,
+		},
+	}
+	charc, err := client.getCharacters() // this is where I want to call the function
+	if err != nil {
+		log.Println("action failed", err)
+		return
+
+	}
+	log.Println(charc)
```

Comment:
You'll want to iterate through the characters to emit their name and description.